pulpcore/pulpcore/app/models/publication.py

```diff
@@ -107,21 +107,36 @@ def delete(self, **kwargs):
             CreatedResource.objects.filter(object_id=self.pk).delete()
             super().delete(**kwargs)
 
+    def update_distributions(self, model=None):
+        """
+        Update distributions with this publication.
```

Comment:
s/Update distributions with this publication./Update distributions associated with this publication's repository and publisher./

Reply:
I think the combination of the summary (first line) and the following paragraph makes this clear.